web/admin/package.json

```diff
@@ -8,6 +8,7 @@
     "@doubledutch/admin-client": "^2.0.2",
     "@doubledutch/firebase-connector": "^4.1.0",
     "@doubledutch/react-components": "^0.1.0",
+    "@doubledutch/react-csv": "^1.1.0",
```

Comment:
please `yarn remove react-csv` since we are replacing w/ `@doubledutch/react-csv`